charts/evm-stack/Chart.lock

```diff
@@ -21,4 +21,4 @@ dependencies:
   repository: https://blockscout.github.io/helm-charts
   version: 1.6.2
 digest: sha256:c849edbcf6b12442d9ec8ffe2f21ceb4f50e58a65c2186340cae6df034f34eae
-generated: "2024-10-27T10:49:54.580428-07:00"
+generated: "2024-10-29T17:33:57.517611+01:00"
```

Comment:
After the reviews and before merging to main I am going to squash all commits and drop this hunk from the patch.

I needed to regenerate this lock file because I rolled back the patch versions of all the charts that were changed in a previous version of this PR. This regen also updated the timestamp.

Reply:
I `git checkout <from-fork> -- charts/evm-stack/Chart.lock` so that the linter does not trigger on this.